Products/product.css

```diff
@@ -0,0 +1,38 @@
+@import url("/style.css");
+.filterList {
+    padding: 3rem 0 0 3rem;
+    flex-basis: 250px;
+    min-width: 250px;
+    height: 100vh;
```

Comment:
use rem instead of px

Reply:
Actually.. The point of flex-basis is to have a fixed width( not to rely on root font-size). So, I thought pixels would be the best. Please suggest an alternate way, if there is any?